pytermgui/markup/parsing.py

```diff
@@ -380,6 +381,9 @@ def parse_alias(
 def parse_clear(token: ClearToken, _: ContextDict, get_full: Callable[[], str]) -> str:
     """Parses a clearer token."""
 
+    if token.value == "/~":
+        return "\x1b]8;;\x1b\\"
+
```

Comment:
This is the sequence that shows up and AFAIK this is about the only place it's hard-coded, so it's a bit suspicious as well.

(though sometimes it comes up as the full form, other times only `8;;\x1b\\`?)